financeager/cli.py

```diff
@@ -35,16 +35,15 @@ def main():
     sys.exit(run(**_parse_command()))
 
 
-def run(command=None, config=None, verbose=False, **params):
-    """High-level API entry point.
-    All 'params' are passed to 'Client.safely_run()'.
-    'config' specifies the path to a custom config file (optional). If 'verbose'
-    is set, debug level log messages are printed to the terminal.
-
-    This function can be used for scripting. Provide 'command' and 'params'
-    according to what the command line interface accepts (consult help via
-    `financeager [command] --help`), e.g. {"command": "add", "name":
-    "champagne", "value": "99"}.
+def run(command=None, config_filepath=None, verbose=False, **cl_kwargs):
+    """High-level API entry point, useful for scripts.
+    Run 'command' passing 'cl_kwargs' according to
+    what the command line interface accepts
+    (consult help via `financeager [command] --help`),
+    e.g. {"command": "add", "name":"champagne", "value": "99"}.
+    All kwargs are passed to 'communication.run()'.
+    'config_filepath' specifies the path to a custom config file (optional).
+    If 'verbose'is set, debug level log messages are printed to the terminal.
```

Comment:
Can you re-check this for spaces? E.g. between name and champagne, and verbose and is.